packages/cdk8s-plus/src/imports/k8s.ts

```diff
@@ -5475,6 +5592,28 @@ export interface ReplicaSetSpec {
 
 }
 
+/**
+ * ReplicaSet ensures that a specified number of pod replicas are running at any given time.
+ *
+ * @schema io.k8s.api.apps.v1.ReplicaSet
+ */
+export interface ReplicaSet {
```

Comment:
Ha - since now the generated constructs don't register with any existing names, we have duplicate interfaces (i.e `ReplicateSet` and `RepliateSetOptions`).

As we discussed, seems like a bug

Reply:
Fixed.